contrib/lua/premake5.lua

```diff
@@ -21,8 +21,8 @@ project "lua-lib"
 		"etc/*.c"
 	}
 
-	configuration "linux or bsd or hurd or aix"
+	filter "system:linux or system:bsd or system:hurd or system:aix"
```

Comment:
Oh good to know, I don't like that line that much either, it's shown on the wiki but not actually "documented" as a feature...